### DISCUSSION TITLE
Αναβολή και αναπλήρωση μαθήματος με δήλωση παρουσίας

### DISCUSSION THREAD

**Discussion by epidrome** [courses-ionio/cscw#170] [Announcements]
Το μάθημα της Δευτέρας και Τρίτης 13-14 Φεβρουαρίου αναβάλλεται και θα αναπληρωθεί την επόμενη εβδομάδα.

Στο μεταξύ θα πρέπει να ξεκινήσετε την ενασχόληση με τις δραστηριότητες της 1ης εβδομάδας #169 

Πρακτικά το μάθημα γίνεται με απαλλακτική εργασία-ασκήσεις όπως φαίνεται στο αναλυτικό πρόγραμμα. 

Οι ασκήσεις είναι κατά το ήμισυ παρόμοιες με αυτές του προαπαιτούμενου μαθήματος (επικοινωνία ανθρώπου-υπολογιστή). Αυτές οι ασκήσεις μπορούν να γίνουν αυτόνομα, αλλά ειδικά για τις νέες ασκήσεις θα πρέπει να γίνουν [τουλάχιστον δύο συναντήσεις για την παρουσίαση της προόδου σας.](https://epidrome.github.io/teaching/classroom/).

Για τα ραντεβού των συναντήσεων του μαθήματος δεν θα χρησιμοποιήσουμε το calendly, αλλά τα σχόλια παρακάτω, όπου μπορείτε να δηλώνετε την παρουσία σας στην αντίστοιχη μέρα-ώρα με τα στοιχεία σας.

Στις συναντήσεις του μαθήματος γίνεται παρουσίαση της δραστηριότητας σας και συζήτηση. Οι απορίες γίνονται αποκλειστικά στην αντίστοιχη [περιοχή της βοήθειας](https://github.com/courses-ionio/cssw/discussions/q-a) έτσι ώστε όλοι να έχουν πρόσβαση κάθε στιγμή και από κάθε τοποθεσία.

**Comment by epidrome** [comment]
Δευτέρα 20 Φεβρουαρίου 6μμ

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | Π2016072 | |
| 2 | inf2021013 | |
| 3 | inf2021055 |  |

**Reply by AlxiKan** [reply]
Καλησπέρα, θα ήθελα να έρθω στο μάθημα της δευτέρας στις 18:00.

Αλεξιος Κανταρτζής Π2016072.

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία στο μάθημα της δευτέρας 18:00

Νικόλαος Αναγνωστόπουλος, inf2021013

**Reply by Axileaszervos** [reply]
Καλησπέρα, θα ήθελα να δηλώσω παρουσία στο μάθημα της δευτέρας στις 18:00.

Ζερβός Αχιλλέας/inf2021055.

**Comment by epidrome** [comment]
~~Τρίτη 21 Φεβρουαρίου 12μμ~~

| # | ΑΜ | GitHub profile |
| -- | -- | -- |
| 1 | 2003001 | gh link |
| 2 | 2003002 | gh link |
| 3 | ... | ... |
